client/asset/btc/btc.go

```diff
@@ -422,8 +445,20 @@ func (btc *ExchangeWallet) Fund(value uint64, nfo *dex.Asset) (asset.Coins, erro
 	var spents []*output
 	fundingCoins := make(map[string]*compositeUTXO)
 
+	// TODO: We are calculating required funds assuming segwit (discounted
+	// witness), but existing utxos may not be witness and thus we should be
+	// counting total input bytes rather than vbytes. Also, even for the chained
+	// swaps, can we make sure that the change outputs that fund further swaps
+	// P2WPKH or is this up to the wallet software???
```

Comment:
Poorly phrased with "bytes rather than vbytes".  But good point that the `input.VBytes()` method handles this.

I'll revise this comment with a note about generating p2wpkh outputs.